content/en/docs/reference/kubernetes-api/ref/workloads/ephemeralcontainer-.md

```diff
@@ -0,0 +1,552 @@
+---
+api_metadata:
```

Comment:
Hi @feloy. Is there a reason why some of the files end with a `-`?

Reply:
The empty apiVersion means the core one

Reply:
The empty string after '-' is for the empty apiVersion. I'm not sure I want to handle this specific case to remove the hyphen for the core API resources

Reply:
This seems important because changing the URI later is quite a bit more of a pain.

Is `content/en/docs/reference/kubernetes-api/ref/workloads/ephemeralcontainer-core.md` easier?

Reply:
Yes, I finally made changes to get hyphenated file names without this trailing `-`

https://deploy-preview-23294--kubernetes-io-master-staging.netlify.app/docs/reference/kubernetes-api/workloads-resources/ephemeral-container/